Marlin/language_it.h

```diff
@@ -184,12 +189,16 @@
 #define MSG_INFO_PROTOCOL                   "Protocollo"
 #if LCD_WIDTH > 19
   #define MSG_INFO_PRINT_COUNT              "Contat. stampa"
-  #define MSG_INFO_FINISHED_PRINTS          "Finiti        "
-  #define MSG_INFO_PRINT_TIME               "Tempo totale  "
+  #define MSG_INFO_COMPLETED_PRINTS         "Completati"
+  #define MSG_INFO_PRINT_TIME               "Tempo totale"
+  #define MSG_INFO_PRINT_LONGEST            "Lavoro piu lungo"
```

Comment:
I don't think we have a `ù` character for Italian currently, but try adding the following line at the top of `language_it.h` and see if it works:
```cpp
#define MAPPER_C2C3  // because of "ù"
```
Note that this will only appear on Hitachi LCD displays that have `WESTERN` extended characters. Most of these displays come rather with `JAPANESE` extended characters.